packages/flutter_tools/test/general.shard/windows_version_validator_test.dart

```diff
@@ -300,10 +96,7 @@ OS 版本:          10.0.22621 暂缺 Build 22621
     );
     final Iterable<RegExpMatch> matches = regex.allMatches(testStr);
 
-    expect(
-      matches.length,
-      3,
-      reason: 'There should be only two matches for the pattern provided',
-    );
+    expect(matches.length, 5,
+        reason: 'There should be only ${matches.length} matches for the pattern provided');
```

Comment:
Isn't this the other way, that we're checking for the constant number, not the actual?

Also, did the number change from 3 to 5 because our regex pattern should have been matching 5 of these lines all along?

Reply:
Ah yes, I made a mistake with the reason text by interpolating, it should be hard coded to be 5 matches. And the number changed from 3 to 5 because one, other contributors added lines that were valid with our previous regex pattern and two, we have a different regex pattern now that only looks for valid semvar formatting. And in the `testStr`, we have 5 valid semvar lines that match our pattern.

Previously, we used to look for `OS Version: ...` at the beginning of the line because that was how `systeminfo` stdout was formatted. With `ver`, it has become simpler.